src/main/java/me/tomsdevsn/hetznercloud/objects/request/LoadBalancerDeleteServiceRequest.java

```diff
@@ -1,22 +1,15 @@
 package me.tomsdevsn.hetznercloud.objects.request;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
-import me.tomsdevsn.hetznercloud.objects.general.LBService;
-import me.tomsdevsn.hetznercloud.objects.general.LBTarget;
-
-import java.util.List;
-import java.util.Map;
+import lombok.RequiredArgsConstructor;
 
 @Data
 @Builder
-@AllArgsConstructor
-@JsonInclude(JsonInclude.Include.NON_NULL)
+@RequiredArgsConstructor
 public class LoadBalancerDeleteServiceRequest {
 
     @JsonProperty("listen_port")
-    private long listenPort;
+    private final long listenPort;
```

Comment:
I am not quite sure if we need to set it as `final`.

Also, please use the `Long` class instead of data type `long` (capitalize `L`). Using the data type made some weird issues in the past.